### ISSUE TITLE
`Alman = false` in combination with long char count in line breaks the `{` into a new row

### ISSUE THREAD

**Issue by tscholze** [nicklockwood/SwiftFormat#1534]
Is it configurable that if the style `alman = false` is applied, that the opening bracket on a method will still be on the same line as the function itself?

This is no problem if the function name is short, but if you have some `AppDelegate` methods, the alman style will not be applied.

**Comment by nicklockwood** [comment]
If I understand correctly, you want something like this?

```swift
func foo(bar,
         baz) {
    ...
}
```

Instead of this?

```swift
func foo(bar,
         baz)
{
    ...
}
```

If so, there's current no option for that, however this style is supported if that would work for you?

```swift
func foo(
    bar,
    baz
) {
    ...
}
```

**Comment by nicklockwood** [comment]
Ah, OK. That could be a bug - I'll check.

**Comment by tscholze** [comment]
Thanks buddy! Your work is amazing!

**Comment by nicklockwood** [comment]
@tscholze I found the issue. For some reason I don't recall, the code was set not to unwrap braces if the width would exceed 100 chars. I'll remove that limit, but in the meantime you can override it by setting `--maxwidth 120` (or whatever).

**Comment by nicklockwood** [comment]
@tscholze fixed in 0.52.5. Thanks for the report!

**Comment by tscholze** [comment]
Thanks! ❤️
